docs/v_2.1.0.md

```diff
@@ -1,25 +1,179 @@
+generators.  It also introduced
+[transformer configs] as a way to fine tune the
+targets of transformations (e.g. to which fields
+_selectors_ should be added).  Most of the feature
+requests for kustomize revolve around changing the
+behavior of the builtin generators and
+transformers.
+
+[v2.1.0] adds a _alpha_ plugin framework, that
+encourages users to write their own generators or
+transformers, _declaring them as kubernetes
+objects just like everything else_, and apply them
+as part of the `kustomize build` process.
+
+To inform the API exposed to plugins, and to
+confirm that the plugin framework can offer plugin
+authors the same capabilities as builtin
+operations, all the builtin generators and
+tranformers have been converted to plugin form
+(with a few exceptions awaiting Go module
+refinements).  This means that adding, say, a
+`secretGenerator` or `commonAnnotations` directive
+to your kustomization will (in [v2.1.0]) trigger
+execution of
+[code committed as a plugin](../plugin/builtin).
+
+For more information, see the
+[kustomize plugin documentation].
+
+## Remove load restrictions
+
+![removed training wheels][imgWheels]
+
+The following usage:
+
+```
+kustomize build --load_restrictions none $target
+```
+
+allows a `kustomization.yaml` file used in this
+build to refer to files outside its own directory
+(i.e. outside its [root]).
+
+This is an opt-in to suppress a security feature
+that denies this precise behavior.
+
+This feature should only be used to allow multiple
+overlays (e.g. prod, staging and dev) to share a
+patch file.  To share _resources_, use a relative
+path or URL to a kustomization directory in the
+`resources` directive.
+
+
+## Field changes / deprecations
+
+* Generalized `resources` field.
+
+  The `resources` field has been generalized, and
+  can now accept what formerly could only
+  be specified in the `bases` field.
+
+  This change was made so that the `resources`,
+  `generators` and `transformers` fields all
+  accept the same argument format.
+
+  Each field's argument is a _string list_, where
+  each entry is either a _resource_ (a relative
+  path to a YAML file) or a [_kustomization_] (a
+  relative path or URL pointing to a directory
+  with a kustomization file).  A kustomization
+  directory used in this context is called a
+  [_base_].
+
+  The `bases` field still works, but is no longer
+  necessary, and will likely be deprecated in the
+  next release.  The _base_ as a concept is as
+  important as ever, it's just that two new fields
+  (`generators` and `tranformers`) and one existing
+  field (`resources`) now accept arguments
+  that were once accepted only by `bases`.
+
+  The fact that the `generators` and
+  `transformers` field accept [bases]
+  (i.e. kustomization directories) and the fact
+  that generator and transformer configuration
+  objects are normal k8s resources means that one
+  can generate or transform a generator or a
```

Comment:
```
generate or transform a generator or a transformer
```
How about
```
customize a generator or a transformer
```

Reply:
well, _customize_  is just a synonym for transform in this case, so it's shorter, but doesn't include the notion of _generating_ a generator or transformer.